dev/README.md

```diff
@@ -1,70 +1,61 @@
 # Development Tomcat Environment
 
+It is possible to use Docker containers to test your code, but it is normally more convenient to simply run the repository webapp (**`alfresco.war`**) and Share webapp (**`share.war`**) in a tomcat instance. Options are also available to apply selected AMPs.
 
-It is possible to use Docker containers to test your code, but it is normally more convenient to simply run the
-repository webapp (`alfresco.war`) and Share webapp (`share.war`) in a tomcat instance. Options are also available to
-apply selected AMPs
-
-## Setting up your development environment
+## 1. Setting up your development environment
```

Comment:
I don't think this should be a numbered list, a lot of these "steps" are not really steps and are optional depending on the use-case or people may prefer to work in a different way. I think the previous approach with them just being regular paragraphs was more true to the nature of this documentation.